esphome/components/emporia_vue/emporia_vue.cpp

```diff
@@ -8,114 +9,131 @@ namespace emporia_vue {
 
 static const char *const TAG = "emporia_vue";
 
-EmporiaVueComponent *global_emporia_vue_component;  // NOLINT(cppcoreguidelines-avoid-non-const-global-variables)
+EmporiaVueComponent *global_emporia_vue_component =
+    nullptr;  // NOLINT(cppcoreguidelines-avoid-non-const-global-variables)
 
 void EmporiaVueComponent::dump_config() {
   ESP_LOGCONFIG(TAG, "Emporia Vue");
   LOG_I2C_DEVICE(this);
-
-  // TODO: Log phases
-  for (PhaseConfig *phase : this->phases_) {
-    //LOG_SENSOR("  ", "Phase", phase);
+  ESP_LOGCONFIG(TAG, "  Sensor Poll Interval: %dms", this->sensor_poll_interval_);
+
+  for (auto *phase : this->phases_) {
+    std::string wire;
+    switch (phase->get_input_wire()) {
+      case PhaseInputWire::BLACK:
+        wire = "BLACK";
+        break;
+      case PhaseInputWire::RED:
+        wire = "RED";
+        break;
+      case PhaseInputWire::BLUE:
+        wire = "BLUE";
+        break;
+    }
+    ESP_LOGCONFIG(TAG, "  Phase Config");
+    ESP_LOGCONFIG(TAG, "    Wire: %s", wire.c_str());
+    ESP_LOGCONFIG(TAG, "    Calibration: %f", phase->get_calibration());
+    LOG_SENSOR("    ", "Voltage", phase->get_voltage_sensor());
   }
 
-
-  for (CTSensor *ct_sensor : this->ct_sensors_) {
+  for (auto *ct_sensor : this->ct_sensors_) {
     LOG_SENSOR("  ", "CT", ct_sensor);
-    // TODO: Log other details
+    ESP_LOGCONFIG(TAG, "    Phase Calibration: %f", ct_sensor->get_phase()->get_calibration());
+    ESP_LOGCONFIG(TAG, "    CT Port Index: %d", ct_sensor->get_input_port());
   }
 }
 
 void EmporiaVueComponent::setup() {
   global_emporia_vue_component = this;
 
-  this->i2c_data_queue_ = xQueueCreate(1, sizeof(EmporiaSensorData));
-  xTaskCreatePinnedToCore(&EmporiaVueComponent::i2c_request_task,
-                          "i2c_request_task",
-                          8192,
-                          nullptr,
-                          0,
-                          nullptr,
-                          1
-  );
+  this->i2c_data_queue_ = xQueueCreate(1, sizeof(SensorReading));
+  xTaskCreatePinnedToCore(&EmporiaVueComponent::i2c_request_task, "i2c_request_task", 4096, nullptr, 0, nullptr, 1);
```

Comment:
I tried changing cores, changing how the delay works, and increasing the delay to >1s, but nothing seemed to fix the OTA update problem except for commenting out this line. 
Stack trace [here](https://gist.github.com/krconv/2336f32c6f2f37c50d03877f5f038796)

Reply:
Mmmmm, wonder if we need to have a handler that halts the core1 task on OTA update. There's other system tasks running on CPU1 so I'm not sure why this one is impacting OTA.

Reply:
I will try to replicate it on my side and decode the stack trace to see if I can pinpoint what's making the IllegalInstruction.